### ISSUE TITLE
Tartube 1.1.0 can't run on Windows 7

### ISSUE THREAD

**Issue by the-robin** [axcore/tartube#27]
Nothing happens when attempting to run on Windows 7. No errors are displayed.

```
ROBN@ROBN-PC C:\Users\ROBN\AppData\Local\Tartube\msys64\home\user\tartube
# ..\..\..\usr\bin\mintty.exe -w hide /bin/env MSYSTEM=MINGW64 /bin/bash -lc /home/user/tartube/tartube_mswin.sh


ROBN@ROBN-PC C:\Users\ROBN\AppData\Local\Tartube\msys64\home\user\tartube
#
```
Running GTK graphics test results in a missing file

```
/home/user/tartube/hello_world_mswin.sh: line 4: cd: tartube: No such file or directory
C:\Users\ROBN\AppData\Local\Tartube\msys64\mingw64\bin\python3.exe: can't open file 'hello_world.py': [Errno 2] No such file or directory
/bin/env: Exit 2.
```
I'm not sure where is is looking. Are there other dependencies I do not have installed?

**Comment by axcore** [comment]
This is fixed in tomorrow's release.

**Comment by axcore** [comment]
Fixed in v1.2.0.
